openhcl/virt_mshv_vtl/src/processor/hardware_cvm/mod.rs

```diff
@@ -1195,6 +1215,73 @@ impl<B: HardwareIsolatedBacking> UhProcessor<'_, B> {
 
         Ok(reprocessing_required)
     }
+
+    fn get_vsm_vp_secure_config_vtl0(
+        &mut self,
+        requesting_vtl: GuestVtl,
```

Comment:
Would it be cleaner to have a target_vtl parameter here, and rename these to not have _vtl0 in the name? If we do someday need to support more than vtls 1 and 0 that feels a bit more generic?